### PR TITLE
Remove the AssetConfig

### PR DESCRIPTION
**Pull request by deads2k**
Remove the AssetConfig

/assign spadgett
/assign mfojtik

@juanvallejo

### CODE REVIEW

**Comment by spadgett** [comment]
@sdodson We should be able to remove assetConfig from the openshift-ansible master-config.yaml template when this merges.

@jwforres fyi

**Comment by soltysh** [comment]
> @sdodson We should be able to remove assetConfig from the openshift-ansible master-config.yaml template when this merges.

But it'll be migrated to a separate one? So that the webconsole is properly installed, right?

**Comment by sdodson** [comment]
> But it'll be migrated to a separate one? So that the webconsole is properly installed, right?

It gets migrated to a configmap in openshift-web-console namespace.

**Comment by deads2k** [comment]
@smarterclayton I need a new openshift tag to land this.

**Comment by deads2k** [comment]
larger hammer acquired.

@smarterclayton after this one merges, I'll need yet another tag.

File: pkg/oc/bootstrap/docker/openshift/helper.go [hunk 1]
```diff
@@ -844,11 +836,6 @@ func (h *Helper) updateConfig(configDir string, opt *StartOptions) error {
 			Configuration: &configapi.DefaultAdmissionConfig{Disable: false},
 		}
 
-		if cfg.AssetConfig == nil {
-			cfg.AssetConfig = &configapi.AssetConfig{}
-		}
-		cfg.AssetConfig.ExtensionScripts = append(cfg.AssetConfig.ExtensionScripts, serviceCatalogExtensionPath)
-
```
Comment (spadgett) [inline pkg/oc/bootstrap/docker/openshift/helper.go]: Lines for extensions below this should also be removed (839-849)

**Pull request merged**